extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -103,7 +104,8 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     }
   }
 
-  private everything = () => {
+  private everything = async () => {
+    this.tabId = await BrowserMsg.requiredTabId();
```

Comment:
I feel like `everything()` is the wrong place for setting `this.tabId` because it'll be assigning it every second. Is there a better place for this line?

Reply:
That's right, see https://github.com/FlowCrypt/flowcrypt-browser/pull/2552#discussion_r378788507